app/classifier/tasks/survey/choice.cjsx

```diff
@@ -52,7 +52,7 @@ module.exports = React.createClass
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
If you add `annotation` to the default props, then you don't need to set a default for answers here.